eng/config/PublishData.json

```diff
@@ -1,9 +1,68 @@
+      "Microsoft.CodeAnalysis.Workspaces.MSBuild" : "arcade",
+      "Microsoft.CodeAnalysis.CSharp" : "arcade",
+      "Microsoft.CodeAnalysis.CSharp.CodeStyle" : "arcade",
+      "Microsoft.CodeAnalysis.CSharp.Features" : "arcade",
+      "Microsoft.CodeAnalysis.CSharp.Scripting" : "arcade",
+      "Microsoft.CodeAnalysis.CSharp.Workspaces" : "arcade",
+      "Microsoft.CodeAnalysis.VisualBasic" : "arcade",
+      "Microsoft.CodeAnalysis.VisualBasic.CodeStyle" : "arcade",
+      "Microsoft.CodeAnalysis.VisualBasic.Features" : "arcade",
+      "Microsoft.CodeAnalysis.VisualBasic.Workspaces" : "arcade",
+      "Microsoft.CodeAnalysis.Build.Tasks" : "arcade",
+      "Microsoft.Net.Compilers" : "arcade",
+      "Microsoft.Net.Compilers.Toolset" : "arcade",
+      "Microsoft.NETCore.Compilers" : "arcade",
+      "Microsoft.CodeAnalysis.Debugging" : "arcade",
+      "Microsoft.CodeAnalysis.PooledObjects" : "arcade",
+      "Microsoft.CodeAnalysis.Features" : "arcade",
+
+      "Microsoft.CodeAnalysis.EditorFeatures" : "vssdk",
+      "Microsoft.CodeAnalysis.EditorFeatures.Common" : "vssdk",
+      "Microsoft.CodeAnalysis.EditorFeatures.Text" : "vssdk",
+      "Microsoft.VisualStudio.LanguageServices" : "vssdk",
+      "Microsoft.CodeAnalysis.CSharp.EditorFeatures" : "vssdk",
+      "Microsoft.CodeAnalysis.InteractiveHost" : "vssdk",
+      "Microsoft.CodeAnalysis.VisualBasic.EditorFeatures" : "vssdk",
+      "Microsoft.CodeAnalysis.EditorFeatures.Wpf" : "vssdk",
+
+      "Microsoft.VisualStudio.IntegrationTest.Utilities" : "vs-impl",
+      "Microsoft.CodeAnalysis.LanguageServer.Protocol" : "vs-impl",
+      "Microsoft.CodeAnalysis.Lsif.Generator" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.Apex" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.CodeLens" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.Debugger" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.FSharp" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.IntelliTrace" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.ProjectSystem" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.Razor" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.TypeScript" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.UnitTesting" : "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote" : "vs-impl",
+      "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub" : "vs-impl",
+      "Microsoft.CodeAnalysis.Remote.ServiceHub" : "vs-impl",
+      "Microsoft.CodeAnalysis.Remote.Workspaces" : "vs-impl",
+      "Microsoft.VisualStudio.LanguageServices.LiveShare" : "vs-impl",
+      "Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient" : "vs-impl"
+    }
+  },
   "branches": {
     "dev15.9.x-vs-deps": {
       "nugetKind": "PerBuildPreRelease",
       "version": "2.10.*",
-      "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+      "packageFeeds": "default",
```

Comment:
I'm worried about us changing the existing branches. They won't have the updates to the build scripts that you're making here hence they will likely barf on the abscence of the `nuget` entry. Should probably leave these old entries alone and just change the publish for 16.7 going forward.